modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/other/SameVersionAPITestCase.java

```diff
@@ -69,8 +69,11 @@ public SameVersionAPITestCase(TestUserMode userMode) {
 
     @BeforeClass(alwaysRun = true)
     public void setEnvironment() throws Exception {
-        super.init();
-        restAPIPublisher = new RestAPIPublisherImpl();
+        super.init(userMode);
+        String userName = gatewayContextWrk.getContextTenant().getTenantAdmin().getUserNameWithoutDomain();
+        String password = gatewayContextWrk.getContextTenant().getTenantAdmin().getPassword();
+        String tenantDomain = gatewayContextWrk.getContextTenant().getDomain();
+        restAPIPublisher = new RestAPIPublisherImpl(userName, password, tenantDomain);
```

Comment:
Same here. We will not need this. We can use the variables defined in the super class.

Reply:
fixed